tests/validation/tests/v3_api/test_istio.py

```diff
@@ -0,0 +1,266 @@
+import os
+
+import pytest
+import time
+
+from .common import *  # NOQA
+
+namespace = {"p_client": None, "ns": None, "cluster": None, "project": None}
+istio_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                          "istio_files")
+ISTIO_CATAGLOG = os.environ.get('ISTIO_CATAGLOG', "system-library")
```

Comment:
In general, environment variables should be prefixed with `RANCHER_` so they can easily be parameterized from Jenkins -- all env vars prefixed with `RANCHER_` are collected and passed to the test container in Jenkins pipelines.